www/js/diary/infinite_scroll_list.js

```diff
@@ -143,22 +154,34 @@ angular.module('emission.main.diary.infscrolllist',['ui-leaflet',
     $ionicLoading.show({
         template: $translate.instant('service.reading-server')
     });
-    Timeline.readAllConfirmedTrips(currEnd, ONE_WEEK).then((ctList) => {
+    Timeline.readAllCompositeTrips(currEnd, ONE_WEEK).then((ctList) => {
         Logger.log("Received batch of size "+ctList.length);
-        ctList.forEach($scope.populateBasicClasses);
-        ctList.forEach((trip, tIndex) => {
-            trip.nextTrip = ctList[tIndex+1];
-            $scope.labelPopulateFactory.populateInputsAndInferences(trip, $scope.data.manualResultMap);
-            $scope.enbs.populateInputsAndInferences(trip, $scope.data.enbsResultMap);
+        ctList.forEach((ct, i) => {
+          
+          if (ct.confirmed_place) {
+            const cp = ct.confirmed_place;
+            cp.nextEntry = ctList[i + 1];
+            $scope.populateBasicClasses(cp);
+            $scope.labelPopulateFactory.populateInputsAndInferences(cp, $scope.data.manualResultMap);
+            $scope.enbs.populateInputsAndInferences(cp, $scope.data.enbsResultMap);
+            ct.nextEntry = cp;
+          } else {
+            ct.nextEntry = ctList[i + 1];
+          }
+          $scope.populateBasicClasses(ct);
+          $scope.labelPopulateFactory.populateInputsAndInferences(ct, $scope.data.manualResultMap);
+          $scope.enbs.populateInputsAndInferences(ct, $scope.data.enbsResultMap);
         });
-        // Fill places on a reversed copy of the list so we fill from the bottom up
+        console.time("fillTrajectories")
+        // Fill place names and trajectories on a reversed copy of the list so we fill from the bottom up
         ctList.slice().reverse().forEach(function(trip, index) {
             fillPlacesForTripAsync(trip);
-            fillTrajectoriesForTripAsync(trip);
+            fillTrajectoriesForTrip(trip);
         });
+        console.timeEnd("fillTrajectories")
```

Comment:
Are we intending on leaving the `console.time` for the final product?

Reply:
No, we should take this out

Reply:
taken out 5bc36d62b799f10a396a343f940dea5b688118e7